src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -153,6 +153,10 @@ public static DataMarshaler OutputMarshaler
                                 }
                                 return dict;
                             }
+                            else if(PyLong.IsLongType(pyObj))
+                            {
+                                return PyLong.AsLong(pyObj).ToInt64();
+                            }
```

Comment:
@reddyashish is this the fix for the int to double param method resolution issue?

Reply:
curious as well as I thought that was filed as a seperate bug.

Reply:
also ... isn't a `Long` the same as `Int64`?

Reply:
Not for double. This is the case for int64 bit integers. Before this only int32 were working. 
I will be filing the issue for BigInteger marshaling.